packages/react-native/Libraries/StyleSheet/StyleSheetTypes.js

```diff
@@ -34,6 +34,8 @@ export type EdgeInsetsValue = {
   bottom: number,
 };
 
+export type CursorValue = ?('auto' | 'pointer');
```

Comment:
nit: we should avoid making this public type nullable in Flow, but non-nullable in TypeScript.

Reply:
Any opinion about exposing all the w3c cursors vs just the two iOS use? macOS supports many more, so I diff the type in RNM. Windows and Android also support different subsets

Reply:
There are plenty of cases where it does happen, but I usually don't like the pattern of having multiple supported options, where it is not apparent that some silently won't work. I think, further down the line, when we expose more direct web APIs, that will end up being inevitable though.

Related, if we do merge this, we would want an accompanying PR to the website to update the API documentation there.

Reply:
Sounds good.I can prepare an accompanying PR to the website and merge both together, provided Both are approved. I think I've missed the 0.74 cut, so I'm not sure if this would be cherry-pick worthy and the website will include that.

Reply:
Fixed the flow type.

Reply:
@NickGerleman aaand here's the website update. https://github.com/facebook/react-native-website/pull/4033